be/src/formats/csv/csv_file_writer.h

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "formats/csv/converter.h"
+#include "formats/csv/output_stream.h"
+#include "formats/file_writer.h"
+
+namespace starrocks::formats {
+
+struct CSVWriterOptions : FileWriterOptions {
+    std::string column_terminated_by = ",";
+    std::string line_terminated_by = "\n";
+};
+
+// Note(letian-jiang): for hive only
```

Comment:
In my opinion, CSV is a standard format, why is this class only used for Hive?

Reply:
CSV does not have a strict standard. Thus the format works for hive but may not work for other cases. 

The primary purpose of this class is to support hive + csv. So I put a note here as a disclaimer.